service/src/intTest/java/uk/nhs/adaptors/gp2gp/gpc/GetGpcDocumentComponentTest.java

```diff
@@ -73,6 +75,12 @@ public void When_NewAccessDocumentTaskIsStarted_Expect_DatabaseUpdatedAndAddedTo
         assertThat(storageDataWrapper.getType()).isEqualTo(taskDefinition.getTaskType().getTaskTypeHeaderValue());
         assertThat(storageDataWrapper.getResponse()).contains(EhrStatusConstants.DOCUMENT_ID);
 
+        String messageId = updatedEhrExtractStatus.getGpcAccessDocument()
+            .getDocuments()
+            .get(0)
+            .getMessageId();
```

Comment:
What is the value of messageId? I cannot see where this property is ever set.

Reply:
It is generated in "GetGpcDocumentTaskExecutor".

Reply:
nevermind, found it in the update query